crypto/src/main/java/org/cryptimeleon/incentive/crypto/TokenDsidHash.java

```diff
@@ -0,0 +1,38 @@
+package org.cryptimeleon.incentive.crypto;
+
+import org.cryptimeleon.incentive.crypto.model.IncentivePublicParameters;
+import org.cryptimeleon.incentive.crypto.model.Token;
+import org.cryptimeleon.math.hash.impl.SHA256HashFunction;
+import org.cryptimeleon.math.serialization.converter.JSONConverter;
+
+/**
+ * A token hash code similar to git commit hashes.
+ * Basically hex string of hash of dsid json string (to be cryptimeleon-agnostic and work e.g. in a JS frontend).
+ */
+public class TokenDsidHash {
```

Comment:
dont like the name, as far as I get it this class does not represent the hash itself but something that can hash, right?